ssg/constants.py

```diff
@@ -262,7 +262,8 @@
         "cpe:/o:suse:linux_enterprise_server:12",
     ],
     "sle15": [
-        "cpe:/o:suse:linux_enterprise_server:15",
+        "cpe:/o:suse:suse_linux_enterprise_server:15",
+        "cpe:/o:suse:suse_linux_enterprise_desktop:15",
```

Comment:
This CPE change is incorrect. Please keep the original. The desktop cpe can be added, but it needs to be `cpe:/o:suse:linux_enterprise_desktop`